Cargo.toml

```diff
@@ -5,5 +5,5 @@ members = [
 ]
 
 [patch.crates-io]
-ya-runtime-sdk = { git = "https://github.com/golemfactory/ya-runtime-sdk.git", rev = "f542b9219f163a52b9783c2c8c42e3da6dd3eb39" }
+ya-runtime-sdk = { git = "https://github.com/golemfactory/ya-runtime-sdk.git", rev = "0395b0c704ef644d7f0554ac41e319f03b11c068" }
```

Comment:
Updated this version because it is a latest release of `ya-runtime-sdk` and its version of `ya-runtime-api` matches the one used in `ya-runtime-vm` (Cargo.toml line below)
https://github.com/golemfactory/ya-runtime-sdk/commit/db42d835d381d3ec21991c68f849ff9e28a432d1#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87eR1314